plugins/apps/threatbus_misp/threatbus_misp/plugin.py

```diff
@@ -222,16 +222,23 @@ def config_validators() -> List[Validator]:
             "operations": "Either configure the MISP plugin to use ZeroMQ or Kafka, but not both."
         },
     )
+    api_is_present = Validator(f"plugins.apps.{plugin_name}.api", required=True)
```

Comment:
Apparently this variable is unused? https://lgtm.com/projects/g/tenzir/threatbus/rev/pr-e38d032fd21d2511af30345f86468601718b3b1d